### PR TITLE
Add GET Object ACL

### PR DESCRIPTION
**Pull request by leslitagordita**
* Added GET /object-storage/buckets/{clusterId}/{bucket}/object-acl

### CODE REVIEW

**Comment by patthiel** [comment]
the endpoint requires a `name` query string parameter. This accepts the name of the object in the bucket you want to get object ACL's for.. For example:

```shell
$ curl object-storage/buckets/alpha/pats_cool_bucket/object-acl?name=test.txt

```

everything else looks good

**Review by victoriaroan — CHANGES_REQUESTED** [review]
`name` is accepted as a query parameter. It is the name of the object and it is required.

File: openapi.yaml [hunk 1]
```diff
@@ -11246,6 +11246,80 @@ paths:
...
+            application/json:
+              schema:
+                type: object
+                properties:
+                  acl:
+                    type: string
+                    enum:
+                    - private
+                    - public-read
+                    - authenticated-read
```
Comment (victoriaroan) [inline openapi.yaml:11306]: It can also return `public-read-write` and `custom`.

**Review by victoriaroan — CHANGES_REQUESTED** [review]

File: openapi.yaml (hunk 1 above)
Comment (leslitagordita) [reply openapi.yaml:11310]: Updated!

**Review by victoriaroan — APPROVED** [review]

**Pull request merged**